src/components/storage/MountPointMapping.jsx

```diff
@@ -546,9 +546,10 @@ const RequestsTable = ({
     setIsFormValid,
     setStepNotification,
 }) => {
-    const mountPointConstraints = useMountPointConstraints();
     const { diskSelection, partitioning } = useContext(StorageContext);
+    const selectedDisks = diskSelection.selectedDisks;
     const deviceData = useOriginalDevices();
+    const mountPointConstraints = useMountPointConstraints({ deviceData, selectedDisks });
```

Comment:
s/deviceData/devices

This is a named parameter, you can't use arbitrary naming.